cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -305,8 +304,6 @@ def upload_splice_junctions_bed_to_scout(self, dry_run: bool, case_id: str) -> N
                 )
 
             LOG.debug(f"Splice junctions bed file {splice_junctions_bed.path} found")
-            dna_sample_id: str
-            dna_cases: list[str]
```

Comment:
ruff would not allow a commit if these were present, but this could easily be reverted if you want them